tests/integration/Cron/Workers/SendingQueue/SendingQueueTest.php

```diff
@@ -621,10 +620,13 @@ function testItDoesNotSendToInactiveSubscribers() {
   }
 
   function testItPausesSendingWhenProcessedSubscriberListCannotBeUpdated() {
-    $sending_task = Mock::double(SendingTask::create(), [
-      'updateProcessedSubscribers' => false,
-    ]);
-    $sending_task->id = 100;
+    $sending_task = $this->createMock(SendingTask::class);
+    $sending_task
+      ->method('updateProcessedSubscribers')
+      ->will($this->returnValue(false));
+    $sending_task
+      ->method('__get')
```

Comment:
I would be better to add `->with('id')` so that this mocks only `$sending_task->id` calls.